thirdeye-ui/src/app/pages/anomalies-create-page/anomalies-create-page.component.tsx

```diff
@@ -0,0 +1,214 @@
+            );
+            if (searchedAlert) {
+                return searchedAlert;
+            }
+        }
+
+        return null;
+    }, [alerts, selectedAlertId]);
+
+    const handleCancelClick = (): void => {
+        navigate(getAnomaliesAllPath());
+    };
+
+    const handleAnomalyCreate = (editableAnomaly: EditableAnomaly): void => {
+        createAnomaly(editableAnomaly)
+            .then((data) => {
+                notify(
+                    NotificationTypeV1.Success,
+                    t("message.create-success", { entity: t("label.anomaly") })
+                );
+                // TODO: Remove
+                window.open(getAnomaliesViewPath(data.id), "_blank");
+            })
+            .catch((error) => {
+                notifyIfErrors(
+                    ActionStatus.Error,
+                    getErrorMessages(error),
+                    notify,
+                    t("message.create-error", {
+                        entity: t("label.alert"),
+                    })
+                );
+            });
+    };
+
+    const pageHeaderProps: PageHeaderProps = {
+        title: t("label.report-missed-anomaly"),
+        breadcrumbs: [
+            {
+                link: getAnomaliesAllPath(),
+                label: t("label.anomalies"),
+            },
+            {
+                link: getAnomaliesCreatePath(),
+                label: t("label.create"),
+            },
+            ...(selectedAlert
+                ? [
+                      {
+                          link: getAlertsAlertViewPath(selectedAlert.id),
+                          label: selectedAlert.name,
+                      },
+                  ]
+                : []),
+        ],
+        customActions: (
+            <PageHeaderActionsV1>
+                <Button
+                    color="primary"
+                    size="small"
+                    variant="outlined"
+                    onClick={() => setIsHelpPanelOpen(true)}
+                >
+                    <Box component="span" mr={1}>
+                        {t("label.need-help")}
+                    </Box>
+                    <Box component="span" display="flex">
+                        <Icon
+                            fontSize={24}
+                            icon="mdi:question-mark-circle-outline"
+                        />
+                    </Box>
+                </Button>
+                &nbsp;
+            </PageHeaderActionsV1>
+        ),
+    };
+
+    return (
+        <PageV1>
+            <PageHeader {...pageHeaderProps} />
+            <HelpDrawerV1
+                cards={anomaliesBasicHelpCards}
+                isOpen={isHelpPanelOpen}
+                title={`${t("label.need-help")}?`}
+                onClose={() => setIsHelpPanelOpen(false)}
+            />
+
+            <LoadingErrorStateSwitch
+                isError={alertsStatus === ActionStatus.Error}
+                isLoading={alertsStatus === ActionStatus.Working}
```

Comment:
Change to `alertsStatus === ActionStatus.Working || alertsStatus === ActionStatus.Initialize` or whatever the initial state is

Reply:
Added `Initial` as well